h2o-extensions/xgboost/src/main/java/ml/dmlc/xgboost4j/java/XGBoostRegTree.java

```diff
@@ -71,6 +72,11 @@ public final float getLeafValue(FVec feat, int root_id) {
     return UnsafeUtils.get4f(_nodes, pos + 12);
   }
 
+  @Override
+  public RegTreeImpl.Node[] getNodes() {
```

Comment:
this should be fixed, getNodes should not return a structure defined in `RegTreeImpl` there should be no connection of XGBoostRegTree and RegTreeImpl

Reply:
This was a forgotten leftover. Fixed. Current status is correct:

```java
  @Override
  public RegTreeNode[] getNodes() {
    throw new UnsupportedOperationException();
  }
```